covalent/_file_transfer/strategies/__init__.py

```diff
@@ -20,3 +20,4 @@
 
 from .http_strategy import HTTP
 from .rsync_strategy import Rsync
+from .s3_strategy import S3
```

Comment:
@santoshkumarradha one option is to modify this statement to say:

```
try:
    import boto3
    from .s3_strategy import S3
except:
    # print an error message that this strategy is not supported
```

Reply:
@wjcunningham7 @santoshkumarradha Should I go ahead and make these changes?

Reply:
Yes ! This is a better implementation for now and we can have a backlog issue to pull this off and add as plugin.

Reply:
Updated! Please review if it is proper.

Reply:
@poojithurao , this change needs to be inside S3 (when importing S3), if you change it in [covalent/_file_transfer/strategies/__init__.py](https://github.com/AgnostiqHQ/covalent/pull/907/files/34ad298cd29bebc66ed350d768e504922220e98b#diff-8d7ffc055a750f06342295a4a4568e9d39faa516279b5db1ea4d289aa0ce09c1), It will raise exception when we import other strategies about boto3.

Reply:
this should be the only strategy that needs boto3

Reply:
correct, thats why it should be changed inside S3 stratergy and not in main [covalent/_file_transfer/strategies/init.py](https://github.com/AgnostiqHQ/covalent/pull/907/files/34ad298cd29bebc66ed350d768e504922220e98b#diff-8d7ffc055a750f06342295a4a4568e9d39faa516279b5db1ea4d289aa0ce09c1) (which is the file that we are commenting right now on)